maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -400,12 +402,14 @@ class ApiClient(
     }
 }
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Why this option?

Reply:
Currently any api changes will break the client. This option will future proof such cases